instrumentation/clickhouse-client-0.5/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/clickhouse/ClickHouseClientInstrumentation.java

```diff
@@ -63,7 +64,10 @@ public static void onEnter(
           ClickHouseDbRequest.create(
               clickHouseRequest.getServer().getHost(),
               clickHouseRequest.getServer().getPort(),
```

Comment:
Would leaving out the port number from the connection string cause a similar problem?

Reply:
no,  this was only applicable to the getDatabase() method because it's the only one [thats Optional](https://github.com/ClickHouse/clickhouse-java/blob/main/clickhouse-client/src/main/java/com/clickhouse/client/ClickHouseNode.java#L1035-L1037). The other methods like port and host will use the [defaults](https://github.com/ClickHouse/clickhouse-java/blob/main/clickhouse-client/src/main/java/com/clickhouse/client/ClickHouseNode.java#L110-L115) and should never not have a value